packages/math-inline/src/main.jsx

```diff
@@ -290,7 +297,8 @@ export class Main extends React.Component {
                         instructionsElement = document.createElement('span');
                         instructionsElement.id = instructionsId;
                         instructionsElement.className = classes.srOnly;
-                        instructionsElement.textContent = 'This field automatically displays a math keypad. Both keypad and keyboard input are accepted, and keyboard entry accepts LaTeX markup.';
+                        instructionsElement.textContent =
+                        'This field supports both keypad and keyboard input. Use the keyboard to access and interact with the on-screen math keypad, which accepts LaTeX markup. Use the down arrow key to open the keypad and navigate its buttons. Use the escape key to close the keypad and return to the input field.';
```

Comment:
I am wondering if we should use the translator for this since it's on the player side. Maybe it also needs to support spanish

Reply:
I've added this question in the feedback request for George: https://illuminate.atlassian.net/browse/PD-2453?focusedCommentId=1503466